docs/changes/io.ascii/14369.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+CDS and MRT tables with units that contain with multiple divisions, such as
```

Comment:
Note that I thought this change was worthwhile specifically mentioning under `io.ascii`, since it affects that package most. I have not included a specific test, since I couldn't easily find an existing table that had multiple solidi.

Reply:
What about that table given in https://github.com/astropy/astropy/issues/14355 ?

Reply:
I'd prefer to use an actually published table...  Note that I used the unit that causes the problems in the tests, so the table from #14355 is now fine.